src/virtual-machine.js

```diff
@@ -182,14 +182,16 @@ class VirtualMachine extends EventEmitter {
      * @return {!Promise} Promise that resolves after targets are installed.
      */
     loadProject (input) {
-        if (typeof input === 'object' && !(input instanceof ArrayBuffer)) {
+        if (typeof input === 'object' && !ArrayBuffer.isView(input)) {
```

Comment:
Oh cool - I didn't know about `isView`!

Reply:
@cwillisf Turns out ArrayBuffers are not views of themselves. Just added the instanceof check back in, but the isView check should work for Uint8Array, Float32Array, and all those good things.